skxray/cdi.py

```diff
@@ -109,7 +115,7 @@ def convolution(array1, array2):
 
     Returns
     -------
-    array :
+    arr : array
```

Comment:
I am pretty sure we should remove convolution in favor of http://docs.scipy.org/doc/scipy-0.15.1/reference/generated/scipy.ndimage.filters.gaussian_filter.html'

As we discussed offline, the `truncate` argument controls how many sigma the Gaussian kernel is computed out to, tuning performance vs. accuracy. The `mode` argument might also be relevant. If I understand correctly, the standard in your application is to use `reflect`. In my naive opinion, using `constant` with `cval=0` might be more accurate.